src/site/layouts/tests/landing_page/landing_page.unit.spec.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import { getLayout, parseFixture, renderHTML } from '../../../tests/support';
+
+const layoutPath = 'src/site/layouts/landing_page.drupal.liquid';
+const layout = getLayout(layoutPath);
+
+describe('intro', () => {
+  describe('no fieldTitleIcon', () => {
+    const data = parseFixture(
+      'src/site/layouts/tests/landing_page/fixtures/landing_page.json',
+    );
+
+    it('renders elements with expected values', async () => {
+      const container = await renderHTML(layoutPath, layout, data, {
+        save: true,
+      });
+      expect(container.querySelector('h1').innerHTML).to.equal(data.title);
+      expect(container.querySelector('.va-introtext p').innerHTML).to.equal(
+        data.fieldIntroText,
+      );
+      expect(
+        container.querySelector('i.icon-large.white.hub-icon-foo'),
+      ).to.equal(null);
+    });
+  });
+
+  describe('with fieldTitleIcon', () => {
+    const data = parseFixture(
+      'src/site/layouts/tests/landing_page/fixtures/landing_page_with_icon.json',
+    );
+
+    it('renders fieldTitleIcon', async () => {
+      const container = await renderHTML(layoutPath, layout, data);
+      expect(container.querySelector('h1').innerHTML).to.equal(data.title);
+      expect(container.querySelector('.va-introtext p').innerHTML).to.equal(
+        data.fieldIntroText,
+      );
+      expect(
+        container.querySelector('i.icon-large.white.hub-icon-foo'),
+      ).not.to.equal(null);
+    });
+  });
```

Comment:
Imagine a utility like `executeAxeCheck` -> https://github.com/department-of-veterans-affairs/vets-website/pull/10623/files#diff-fca242d8f94a9b0add72c42c8676727b81504cd1f52eb8a4e898c6992807bcf0R32

```suggestion
  });
  it('passes an axe check-, () => {
    const container = await renderHTML(layoutPath, layout, data);
    executeAxeCheck('https://www.va.gov/something', container);
  });
```